tools/should-skip-ci.js

```diff
@@ -0,0 +1,95 @@
+import { readFileSync } from "fs";
+
+const SKIP_CACTI = "skip-cacti-ci";
```

Comment:
Please add a in-code comment prior to this line specifying that this is the new tag which can be used to facilitate ci-skip

Reply:
@zondervancalvez please update the comment to also include that it is specifically for maintainers. Also rename this file to ci-skip-for-maintainers.js